docs/source/details.rst

```diff
@@ -22,12 +22,20 @@ category type:
 
     df[col] = df[col].astype('category')
 
-When loading, only columns which have dictionary encoding for every row-group,
-and are also included in the optional keyword parameter ``categories`` will
-result in categorical-type columns in the output. Other columns will be
-converted to object type, which is potentially expensive. On the other hand,
-if most entries in the column are NULL (see below), then converting to/from
-category type is probably unnecessary.
+The efficiently load a column as a categorical type, include it in the optional
+keyword parameter ``categories``; however it must be encoded as dictionary
+throughout the dataset (as it will, if written by fastparquet).
+
+.. code-block:: python
+
+    pf = ParquetFile('input.parq)
+    df = pf.to_pandas(categories={'cat': 12})
+
+Where we provide a hint that the column `cat` has up to 12 possible values.
```

Comment:
Note that `cat` does not do the syntax highlighting you're expecting in rst.

I recommend previewing your work here by building locally.  This is usually done by `make html` within `fastparquet/docs`